hydra-node/src/Hydra/Ledger/Cardano.hs

```diff
@@ -221,6 +224,89 @@ utxoFromTx (Tx body@(ShelleyTxBody _ ledgerBody _ _ _ _) _) =
 utxoMin :: Utxo -> Utxo
 utxoMin = Utxo . uncurry Map.singleton . Map.findMin . utxoMap
 
+--
+-- Transaction Construction
+--
+
+type TxBuilder = TxBodyContent BuildTx Era
```

Comment:
I like the name, but introducing this, essentially has readers / editors jump through one more indirection when they would want to find out what that type actually is, e.g. what record fields it has.